pages/workshop22.md

```diff
@@ -68,6 +68,22 @@ Across all of the training modules, we have three overaching goals:
 1. Provide attendees with hands-on practice and demonstration
 1. Help demystify buzz words
 
+## FAQs
+#### How much is supposed to be done at the end of the workshop?
+_We aim to have each module at least 90% complete by the end of the workshop._
```

Comment:
```suggestion
_We aim to have each module 90% complete by the end of the workshop.
Additional refinement and improvement can happen asynchronously afterward the workshop_
```

Reply:
Please feel free to edit it to read however you are comfortable. I just put a number there so we would have something.